src/components/blocks/Header/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Logo from 'components/reusable/Logo/Logo';
+import { IconButton } from '@material-ui/core';
+import HomeIcon from '@material-ui/icons/Home';
+import Avatar from '@material-ui/core/Avatar';
+import { header, headerContent, divider } from './Header.scss';
+import { avatar } from '../../reusable/Avatar/Avatar.scss';
+import avatartSrc from 'assets/images/DG.png';
+
+const logoProps = {
+  text: 'D&G (Dima & Goman)'
+}
+
+const Header = () => (
+    <header className={ header }>
+      <div className={ headerContent }>
+        <IconButton >
+          <HomeIcon fontSize="large"/>
+        </IconButton>
+        <div className={ divider }></div>
+        <Logo logoContent={ logoProps }/>
+      </div>
+      <div className={ avatar }>
+        <Avatar alt="avatar" src={ avatartSrc } />
```

Comment:
Could this Avatar (svg) component take className property? if so - then <div className={ avatar }> is redundant

Reply:
It can, but I need a wrapper for better display.